src/styles/blocks/card.scss

```
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform, 300ms;
+
+  &__image {
+    display: block;
+    margin: auto;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    color: $default-title-color;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+
+    &-count {
+      @include onBlockDisplay;
+      font-weight: $default-font-weight;
+      font-size: 10px;
+      line-height: 14px;
+      text-align: right;
+    }
+  }
```

The `card__reviews-count` it's a separate element. It shouldn't be nested inside the card__reviews
```suggestion
  }
  
  &-reviews-count {
    @include onBlockDisplay;
    font-weight: $default-font-weight;
    font-size: 10px;
    line-height: 14px;
    text-align: right;
  }
```